src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java

```diff
@@ -49,12 +49,6 @@
  */
 public abstract class AbstractLoader
     extends DefaultHandler {
-    /** Feature that enables loading external DTD when loading XML files. */
-    private static final String LOAD_EXTERNAL_DTD =
-        "http://apache.org/xml/features/nonvalidating/load-external-dtd";
```

Comment:
Those features are enabled by [default](https://xerces.apache.org/xerces2-j/features.html). I moved them to the tests - to always be sure that they are still enabled.